### ISSUE TITLE
Accessibility Issue with ellipsis page actions

### ISSUE THREAD

**Issue by Shane-E** [michaelbromley/ngx-pagination#413]
**Angular version**: 16.2.12

**ngx-pagination version**: ^6.0.3

**Description of issue**: When implementing a custom template using the PaginationControlsDirective, there appears to be no way to provide contextual aria-labels for the ellipsis buttons/links that are automatically added to the page array, resulting in screen readers reading the label "..." which is not helpful to a user. I would like to see if there's a way that these ellipsis buttons/links can have a better default aria-label such as "Next Page Set" or "Previous Page Set" corresponding to the ellipsis button that performs that action and/or possibly allow customization of the aria-label for those specific buttons should someone wish to change them.

**Steps to reproduce**: Configure a custom pagination template that has more pages than the maxSize (default maxSize value is 7). Alternatively, go to https://michaelbromley.github.io/ngx-pagination/#/custom-template and interact with the page action buttons with a screen reader

**Expected result**: Ellipses buttons/links should have a more contextually relevant aria-label and be customizable

**Actual result**: Ellipses buttons/links are read as "Page" by screen readers

**Demo**: Interact with the pagination controls using a screen reader at https://michaelbromley.github.io/ngx-pagination/#/custom-template

Any relevant code: https://github.com/michaelbromley/ngx-pagination/blob/master/projects/ngx-pagination/src/lib/pagination-controls.directive.ts#L183-L191
